.github/workflows/ci.yml

```
@@ -0,0 +1,97 @@
+    needs: [frontend-test, backend-test]
+    if: github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+      
+    - name: Build and push Frontend
+      uses: docker/build-push-action@v2
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)